ci/test/06_script_b.sh

```diff
@@ -46,6 +46,18 @@ if [ "${RUN_TIDY}" = "true" ]; then
           " src/policy/settings.cpp"\
           " src/rpc/fees.cpp"\
           " src/rpc/signmessage.cpp"\
+          " src/util/bip32.cpp"\
+          " src/util/bytevectorhash.cpp"\
+          " src/util/error.cpp"\
+          " src/util/getuniquepath.cpp"\
+          " src/util/hasher.cpp"\
+          " src/util/message.cpp"\
+          " src/util/moneystr.cpp"\
+          " src/util/serfloat.cpp"\
+          " src/util/spanparsing.cpp"\
+          " src/util/strencodings.cpp"\
+          " src/util/syserror.cpp"\
+          " src/util/url.cpp"\
```

Comment:
Actually, this PR affects more files, e.g., `asmap.cpp`, `readwritefile.cpp` etc. Should they be listed here as well?

Reply:
IWYU generates spurious output for some .cpp files, which would require adding headers that I'm not sure are correct, so I've excluded them for now.